jobManager.py

```diff
@@ -82,6 +82,9 @@ def __manage(self):
         elif Config.VMMS_NAME == "ec2SSH":
             from vmms.ec2SSH import Ec2SSH
             vmms = Ec2SSH()
+        elif Config.VMMS_NAME == "dockerSSH":
+            from vmms.dockerSSH import DockerSSH
+            vmms = DockerSSH()
 
```

Comment:
Basically something was wrong in my config and it was exiting, but it was hard to pin down where it was going wrong. Printing something to stdout will also be very helpful with that.